### PR TITLE
Bib 779 update api endpoints for pre translate

### PR DESCRIPTION
**Pull request by elisabeth-shakespeare**

### CODE REVIEW

File: src/Aquifer.API/Endpoints/AI/Translate/Endpoint.cs
Comment (elisabeth-shakespeare) [inline src/Aquifer.API/Endpoints/AI/Translate/Endpoint.cs:1]: Doesn't seem right to delete the file.
From what I've seen with fastendpoints, the only way to deprecate an endpoint is to use its versioning and set it to deprecate at a version, and then it's not set up for that version. That didn't seem right either.

File: src/Aquifer.API/Endpoints/Comments/Threads/Create/Endpoint.cs [hunk 1]
```diff
@@ -34,10 +34,19 @@ private async Task<Response> CreateResourceContentVersionCommentThreadAsync(Requ
...
         EndpointHelpers.ThrowErrorIfNull<Request>(resourceContentVersion, x => x.TypeId, "No type found for given id.", 404);
 
+        List<ResourceContentStatus> notAllowedStatuses =
+            [ResourceContentStatus.TranslationAiDraftComplete, ResourceContentStatus.TranslationAwaitingAiDraft];
+
+        if (notAllowedStatuses.Contains(resourceContentVersion!.ResourceContent.Status))
+        {
+            ThrowError(x => x.TypeId, $"Resource for type {req.TypeId} is not in correct status");
+        }
+
```
Comment (elisabeth-shakespeare) [inline src/Aquifer.API/Endpoints/Comments/Threads/Create/Endpoint.cs:49]: I'd like to put this in a Helper Method. Would Common be the best place to put it?

File: src/Aquifer.API/Endpoints/Resources/BibleReferences/Create/Endpoint.cs
Comment (MrGrinst) [inline src/Aquifer.API/Endpoints/Resources/BibleReferences/Create/Endpoint.cs:1]: Looks like there were only formatter changes in this file

File: src/Aquifer.API/Endpoints/Resources/Content/MachineTranslation/Create/Endpoint.cs [hunk 2]
```diff
@@ -17,6 +18,8 @@ public override void Configure()
 
     public override async Task HandleAsync(Request req, CancellationToken ct)
```
Comment (MrGrinst) [inline src/Aquifer.API/Endpoints/Resources/Content/MachineTranslation/Create/Endpoint.cs:19]: As I understand it this endpoint is going away entirely so probably not worth making changes to it

Comment (elisabeth-shakespeare) [reply src/Aquifer.API/Endpoints/Resources/Content/MachineTranslation/Create/Endpoint.cs:19]: Is Retranslate going away too?

File: src/Aquifer.API/Endpoints/Resources/BibleReferences/Create/Endpoint.cs
Comment (elisabeth-shakespeare) [reply src/Aquifer.API/Endpoints/Resources/BibleReferences/Create/Endpoint.cs:1]: yeah, I kept running format project wide, and figured might as well push up cleaner stuff while I pushed up other little housekeeping items

**Review by jwinston-bn — APPROVED** [review]

**Pull request merged**